endpoints/setuid.go

```diff
@@ -101,7 +117,7 @@ func NewSetUIDEndpoint(cfg config.HostCookie, syncersByBidder map[string]usersyn
 		}
 
 		setSiteCookie := siteCookieCheck(r.UserAgent())
-		pc.SetCookieOnResponse(w, setSiteCookie, &cfg, cookieTTL)
+		pc.SetCookieOnResponse(w, setSiteCookie, &cfg.HostCookie, cookieTTL)
```

Comment:
For my own understanding, why was this function call updated to be `cfg.HostCookie`. How does this impact the overall goal of this PR?

Reply:
Ah `cfg` here used to be of type `config.HostCookie`. See the `NewSetUIDEndpoint` function signature on line 31. I changed that to be of type `config.Configuration` so I would have access to the entire configuration which is needed by `GetAccount`. As a result, I now need to pass the `HostCookie` field of the `cfg` into `SetCookieOnResponse` here.